src/patchelf.cc

```diff
@@ -39,6 +41,7 @@
 #include <sys/stat.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <stdlib.h>
```

Comment:
In c++ this should be `#include <cstdlib>`

Reply:
And the above cunistd? Sorry I come from a C background... Probably stdlib isn't needed anymore now I don't malloc() anymore.

Reply:
There is no cunistd, since that one is unix specific.